.gitignore

```diff
@@ -10,3 +10,7 @@ src/files/*
 *.phpproj
 *.sln
 *.phpproj.user
+
+# for docker stuff
+data/
```

Comment:
Data folder can be removed, see other comments.

Reply:
removed in: https://github.com/hashtopolis/server/pull/719/commits/c6439bff8cbc20fd0fb20acc15876d2063dfac28